DDDEastAnglia/Helpers/Security/IOAuthClientInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using DotNetOpenAuth.AspNet;
+using Microsoft.Web.WebPages.OAuth;
+
+namespace DDDEastAnglia.Helpers.Security
+{
+    public interface IOAuthClientInfo
```

Comment:
Wow, ok, that's a big interface. Given that it's defining a wrapper around a static class, it should be über-easy to turn this into a set of [Role Interfaces](http://martinfowler.com/bliki/RoleInterface.html) to better adhere to the Interface Segregation Principle.